src/modules/reading-room/queries/getReadingRooms.ts

```diff
@@ -0,0 +1,32 @@
+import ky from 'ky-universal';
```

Comment:
File naam mag in kebab-case, enkel React specifieke zaken gaan we in PascalCase zetten